src/electron/customCSS.ts

```diff
@@ -0,0 +1,14 @@
+import { readFileSync } from 'fs';
+import { pathExistsSync } from 'fs-extra';
```

Comment:
I think `fs-extra` has `readFileSync` as well - you can combine the 2 import statements into a single line.